downloader/src/main/kotlin/vcs/GitWorkingTree.kt

```diff
@@ -72,7 +72,7 @@ open class GitWorkingTree(
                     val path = "$prefix${walk.path}"
 
                     if (walk.repository == null) {
-                        logger.info {
+                        logger.warn {
```

Comment:
From the "type", this is rather a "chore" than a "fix".

Reply:
fixed